src/web/AccessibilityUtil.ts

```diff
@@ -39,27 +39,27 @@ const roleMap: { [key: string]: string } = {
     [Types.AccessibilityTrait.Status]: 'status',
     [Types.AccessibilityTrait.Dialog]: 'dialog',
     [Types.AccessibilityTrait.Switch]: 'switch'
-}; 
+};
 
 // Map of accesssibility live region to an aria-live property.
-const liveRegionMap: { [key: string]: string } = {
+const liveRegionMap: { [key: string]: 'off' | 'assertive' | 'polite'; } = {
```

Comment:
Since you use this union type in multiple places, it would be good to define it once and use the type name rather than another (duplicate) inline definition.